rdrf/rdrf/templatetags/session_refresh_interval.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag
+def session_refresh_interval():
+    # 15 seconds before session expiration in ms
+    return (settings.SESSION_COOKIE_AGE - 15) * 1000
```

Comment:
If we account for warnings to the user, perhaps make this a fraction of the cookie age.

Reply:
Not sure how that will help. The interval is set so that the session will be extended right before expiring. With the current default value of 15 minutes ( down from the old value of 60 minutes) the session will be extended every 14:45 minutes. If we reduce this to let's say 5 minutes we'd also need to update the number of retries accordingly.

Reply:
Once we give users a warning, they might need more time to react